MetaMask/metamask-extension
# UX Multichain: fixed padding for edit screen

This PR is to remove the extra margin between header and content from all the screens and just adding it to the send edit and swap screen.

* Fixes MetaMask/MetaMask-planning#850 

## Screenshots/Screencaps


https://github.com/MetaMask/metamask-extension/assets/39872794/34ab9214-3514-448e-95ac-6226a4402eef



## Manual Testing Steps

- Go to send and swap screen
- Check the content and header has gap of 16px

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
